modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/view.jsp

```diff
@@ -130,4 +130,19 @@ KBAdminManagementToolbarDisplayContext kbAdminManagementToolbarDisplayContext =
 			</aui:form>
 		</clay:container-fluid>
 	</div>
-</div>
\ No newline at end of file
+</div>
+
+<%
+String kbArticleScheduledSuccessMessage = GetterUtil.getString(MultiSessionMessages.get(renderRequest, "kbArticleScheduledSuccessMessage"));
+%>
+
+<c:if test="<%= Validator.isNotNull(kbArticleScheduledSuccessMessage) %>">
+	<script>
+		Liferay.Util.openToast({
+			message: '<%= kbArticleScheduledSuccessMessage %>',
+			toastProps: {
+				autoClose: 20000,
```

Comment:
`autoClose` is an attribute of the toast, doesn't need to be inside `toastProps` ;-)

Reply:
Ty! I'm gonna and apply it in a separate js file to maintain the KB with the minimum `<script>` tag as possible